website/members/forms.py

```diff
@@ -5,12 +5,20 @@
 from django.core.validators import RegexValidator
 from django.utils.translation import gettext_lazy as _
 
+from thabloid.models.thabliod_user import ThabloidUser
+
 from .models import Profile
 
 
 class ProfileForm(forms.ModelForm):
     """Form with all the user editable fields of a Profile model."""
 
+    receive_thabloid = forms.BooleanField(
+        required=False,
+        label="Receive thabloid",
```

Comment:
We actually don't want to define this field in the `members` app at all. It is already an improvement that it's not in the `members` database anymore, but now we have a dependency from `members` onto `thabloid` (while it can only exist the other way around). 

So this should be dynamically defined. For example, via some registration of `profile fields`